src/macro/eval/evalContext.ml

```diff
@@ -440,6 +441,14 @@ let pop_environment ctx env =
 (* Prototypes *)
 
 let get_static_prototype_raise ctx path =
+	let inits =
+		try
+			let inits = IntMap.find path ctx.reset_static_inits in
+			ctx.reset_static_inits <- IntMap.remove path ctx.reset_static_inits;
+			Some inits
+		with Not_found -> None
+	in
+	Option.may (fun (proto, delays) -> List.iter (fun f -> f proto) delays) inits;
```

Comment:
This additional branching via `Option.may` is unnecessary. Please avoid the whole `let inits =` part and just `List.iter` on the inner `inits` instead.